qa/workunits/rbd/rbd-ggate.sh

```diff
@@ -149,17 +149,20 @@ used=`rbd -p ${POOL} --format xml du ${IMAGE} |
 [ "${used}" -lt "${provisioned}" ]
 
 # resize test
+# resizing the underlying image for a GEOM ggate will stop the
+# ggate process servicing the device. So we can resize and test 
+# the disapearence of the device
```

Comment:
@trociny 
Sort of ... The first one is the general assumption for the test. 
The second is needed because the assumption is not always consistent delivered. And once in a while I still get `rbd-ggate list` return the resized device, even though it should have disappeared..
So I again noted that in the comment.